internal/buildengine/engine.go

```diff
@@ -99,7 +100,7 @@ func WithListener(listener Listener) Option {
 // pull in missing schemas.
 //
 // "dirs" are directories to scan for local modules.
-func New(ctx context.Context, client ftlv1connect.ControllerServiceClient, projectRoot string, moduleDirs []string, options ...Option) (*Engine, error) {
+func New(ctx context.Context, client ftlv1connect.ControllerServiceClient, projectRoot string, moduleDirs []string, buildEnv []string, options ...Option) (*Engine, error) {
```

Comment:
This should be a functional option rather than a required arg, as it's optional.

Reply:
done